index.md

```diff
@@ -0,0 +1 @@
+<h1>##</h1>

```

Comment:
# This is an `<h1>` header, which is the largest

## This is an `<h2>` header

###### This is an `<h6>` header, which is the smallest